src/server/game/AuctionHouseBot/AuctionHouseBot.cpp

```diff
@@ -479,8 +479,8 @@ void AuctionHouseBot::PrepareStatusInfos(AuctionHouseBotStatusInfo& statusInfo)
     {
         statusInfo[i].ItemsCount = 0;
 
-        for (int j = 0; j < MAX_AUCTION_QUALITY; ++j)
-            statusInfo[i].QualityInfo[j] = 0;
+        for (uint32& j : statusInfo[i].QualityInfo)
+            j = 0;
```

Comment:
thats a terrible transformed name, something like `countPerQuality` (see next loop for what it does) would have been better

Reply:
true.